src/main/resources/static/js/MainWindow.js

```diff
@@ -1,670 +1,756 @@
-                        )
-                    )
-                )
-                .append($("<tbody>")
-                )
-            )
-        )
-    );
-
-    $(".selectStreams").last().select2({
-        data: streamData,
-        placeholder: "Choose a Datastream",
-        width: "style",
-        dropdownAutoWidth: true
-    }).trigger("change").on("select2:select", function (e) {
-        var sel = $(this);
-        var parent = sel.parent().parent();
-        var json;
-        if (e.params.data["data-value"]) {
-            json = JSON.parse(e.params.data["data-value"]);
-            sel.find("option:selected").attr("data-value", JSON.stringify(json));
-        } else {
-            json = JSON.parse(sel.find("option:selected").attr("data-value"));
-        }
-        if (json) loadStream(json, parent.find(".streamtable"));
-    });
+	var streams = $("#datastreams");
+	streams
+			.append($("<div>")
+					.attr("class", "datastream")
+					.append(
+							$("<div>")
+									.append(
+											$("<label>").text("Name:").attr(
+													"style",
+													"margin-right:10px"))
+									.append(
+											$("<select>").attr("style",
+													"width: 200px").attr(
+													"name", "streams").attr(
+													"class", "selectStreams"))
+									.append(
+											$("<button>")
+													.attr("class",
+															"btn btn-secondary")
+													.attr("onclick",
+															"removeDatastream(this.parentNode.parentNode)")
+													.attr("style",
+															"width:auto; margin-left:10px")
+													.html(
+															"<span class=\"fas fa-minus\" ></span>")))
+					.append(
+							$("<div>")
+									.attr("class", "streamtable")
+									.append(
+											$("<table>")
+													.attr("class", "table")
+													.append(
+															$("<thead>")
+																	.append(
+																			$(
+																					"<tr>")
+																					.append(
+																							$(
+																									"<th>")
+																									.text(
+																											"Unit"))
+																					.append(
+																							$(
+																									"<th>")
+																									.text(
+																											"Column"))))
+													.append($("<tbody>")))));
+
+	$(".selectStreams").last().select2({
+		data : streamData,
+		placeholder : "Choose a Datastream",
+		width : "style",
+		dropdownAutoWidth : true
+	}).trigger("change").on(
+			"select2:select",
+			function(e) {
+				var sel = $(this);
+				var parent = sel.parent().parent();
+				var json;
+				if (e.params.data["data-value"]) {
+					json = JSON.parse(e.params.data["data-value"]);
+					sel.find("option:selected").attr("data-value",
+							JSON.stringify(json));
+				} else {
+					json = JSON.parse(sel.find("option:selected").attr(
+							"data-value"));
+				}
+				if (json)
+					loadStream(json, parent.find(".streamtable"));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['loadStream' was used before it was defined.](https://app.codacy.com/gh/FraunhoferIOSB/ChillImport/pullRequest?prid=4781409)